README.md

```diff
@@ -101,6 +101,34 @@ public function BetterNavigatorEditLink()
 
 (This example uses [sunnysideup/cms_edit_link_field](https://github.com/sunnysideup/silverstripe-cms_edit_link_field) to automatically find an edit link for a specified DataObject, but you can return any URL.)
 
+## Overriding whether better navigator should be shown or not
```

Comment:
This sounds exactly the same as the `ShowBetterNavigator` method that we already have 😄 see https://github.com/jonom/silverstripe-betternavigator#disabling-the-navigator

Reply:
Your correct I have made a change to the default `showBetterNavigator()` that instead has it call `$this->isAPage()` which solves what my code was apparently doing lol. I did adjust the wording of the section you pointed out in the readme to be clear that it can be used to control display in general.